isic/ingest/tests/factories.py

```diff
@@ -83,3 +83,5 @@ class Meta:
     phi_check = factory.Faker('boolean')
     duplicate_check = factory.Faker('boolean')
     lesion_check = factory.Faker('boolean')
+
+    metadata = factory.Dict({})
```

Comment:
@brianhelba This is causing every the tests to fail, but works for `test_core_api_image_ages_are_always_rounded` where I set `accession__metadata__age`. Have you done this any other way?

Reply:
fixed.